src/app/modules/audience/views/engagement/syndication/syndication.component.ts

```diff
@@ -259,7 +259,8 @@ export class SyndicationComponent extends EngagementBaseReportComponent {
 
   public _openLink(data): void {
     if (data && data.referrer) {
-      window.open(data.referrer, '_blank');
+      const link = data.referrer.startsWith('http') ? data.referrer : `https://${data.referrer}`;
```

Comment:
seems that startsWith is [not supported by IE11](https://www.w3schools.com/jsref/jsref_startswith.asp). I'm not sure we have a polyfill for that. Lets use `indexOf('http') !== 0`